partner_invoicing_mode_at_shipping/migrations/16.0.1.1.0/pre-migrate.py

```diff
@@ -1,13 +0,0 @@
-# Copyright 2023 ACSONE SA/NV
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from openupgradelib import openupgrade
-
-# pylint: disable=odoo-addons-relative-import
-from odoo.addons.partner_invoicing_mode_at_shipping.hooks import (
-    _add_one_invoice_per_shipping,
-)
-
-
-@openupgrade.migrate()
-def migrate(env, version):
-    _add_one_invoice_per_shipping(env)
```

Comment:
This migration script remains valid for people migrating from 16 to 18 using the Odoo enterprise migration, which does not consider custom or third party modules for intermediate versions. I suggest we keep it.